src/scheduler/site_code.c

```diff
@@ -3258,7 +3258,7 @@ reconcile_share_tree(share_info *root, share_info *def, int cnt)
 						"%s share for %s too small for children: %d < %d",
 						root->name, shr_class_name_by_idx(i),
 						gross, c_sum);
-					schdlog(PBSEVENT_SCHED, PBS_EVENTCLASS_FILE,
+					log_event(PBSEVENT_SCHED, PBS_EVENTCLASS_FILE,
```

Comment:
Can this be log_eventf and we can get rid of sprintf as well?

Reply:
This is NAS site code.  I chose to not convert it.  I am not going to test it, so it is probably best to not convert it.  I had to change the function name since schdlog() no longer exists.